lua/nvim-treesitter/incremental_selection.lua

```diff
@@ -138,16 +139,14 @@ function M.attach(bufnr)
   local config = configs.get_module "incremental_selection"
   for funcname, mapping in pairs(config.keymaps) do
     if mapping then
-      ---@type string, string|function
-      local mode, rhs
-      if funcname == "init_selection" then
-        mode = "n"
-        ---@type function
-        rhs = M[funcname]
-      else
-        mode = "x"
-        rhs = M[funcname] ---@type function
+      local mode = funcname == "init_selection" and "n" or "x"
+      local rhs = M[funcname] ---@type function
+
+      if not rhs then
+        utils.notify("Unknown keybinding: " .. funcname, vim.log.levels.ERROR)
+        return
```

Comment:
Isn't `return` too aggressive? Keymapping sets in the previous iterations of the for loop will still remain. I suggest: just skip `vim.keymap.set` (putting in the `else` block) or continue. Oh well, it's Lua :) ~~`::continue::`~~

Reply:
I mean, at this point the module configuration is just wrong. We can have a more "graceful" setup but things are going to be whack anyway...

Reply:
Yeah that also sounds good, thanks!

Reply:
Thanks to YOU <3